Gemfile

```diff
@@ -0,0 +1,10 @@
+source "http://rubygems.org"
+
+group :development do
+  gem 'therubyracer'
+  gem 'jekyll'
+  gem 'kramdown'
+  gem 'pygments.rb'
+  gem 'rake'
+  gem 'html-proofer'
+end
```

Comment:
Its possible it doesn't like that we are bundling from another directory or that it is trying to bundle by default for a ruby project?